Dockerfile

```diff
@@ -0,0 +1,69 @@
+####################################################
+# START GLOBAL DECLARATION
+####################################################
+ARG REPO_NAME_DEFAULT=metastore2
+ARG REPO_PORT_DEFAULT=8040
+ARG SERVICE_ROOT_DIRECTORY_DEFAULT=/spring/
+####################################################
+# END GLOBAL DECLARATION
+####################################################
+
+####################################################
+# Building environment (java & git)
+####################################################
+FROM openjdk:11-stretch AS build-env-java
```

Comment:
I am successfully using `openjdk:11-buster` for the PIT service and the indexer within the testbed. It's just the base-os version, but maybe as we are at it anyway, should we update?

Reply:
:+1:  We should use the same base everywhere. Then we have it easier to keep the updates up to date.